Packages/com.chisel.editor/Chisel/Editor/Components/Base/Editor/ChiselHierarchyWindowManager.cs

```diff
@@ -0,0 +1,40 @@
+﻿using UnityEngine;
```

Comment:
This file is for handling the items that show in the hierarchy. It allows us to put an icon behind it that shows what kind of generator it is & if it's additive, subtractive or intersecting. Right now it shows the same icons for all generators, but this can be changed in the future if we want